apps/src/applab/applab.js

```diff
@@ -666,6 +674,17 @@ Applab.init = function(config) {
     }
     drawDiv();
 
+    // Ignore the user's levelHtml for levels without design mode. levelHtml
+    // should never be present on such levels, however some levels do
+    // have levelHtml stored due to a previous bug. HTML set by levelbuilder
+    // is stored in startHtml, not levelHtml.
```

Comment:
Good comment :) I started asking about this, then reread the comment, and realized it was already answer for me :)